src/libraries/Microsoft.CSharp/ref/Microsoft.CSharp.cs

```diff
@@ -9,16 +9,27 @@ namespace Microsoft.CSharp.RuntimeBinder
     [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
     public static partial class Binder
     {
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Types and members might be removed")]
```

Comment:
These messages should be updated.

cc @safern - shouldn't API Compat be failing here? The messages don't match between `src` and `ref`.

Reply:
From reading  the code it should've done it. I'm not sure why it didn't flag it in this case. However now the new API Compat is moving further along I don't think it's worth fixing this in the old API Compat.

Reply:
Sounds good to me. @joperezr and I were unsure if the new API Compat was in or not, and thought it might be a bug with the new code.

Reply:
Makes sense, I appreciate cc'ing me and asking. The new API Compat is not in yet. I expect it to be in around Preview4/5 days.